### PR TITLE
Fortune cookies

### PR DESCRIPTION
**Pull request by Neon72**
a few more quotes for fortune cookies

## Description
Added a few quotes to fortune cookies to pull from

## Changes
6 new lines of dialogue for Fortune cookies

## Related Issues
None

## Checklist

- [ ] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [ ] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [x] I followed the existing code standards and didn't mess up the formatting.
- [x] I did my best to add documentation to any public classes or methods I added.
- [ ] I added sufficient Unit Tests to cover my code.

### CODE REVIEW

File: src/main/resources/languages/messages_en.yml [hunk 1]
```diff
@@ -155,6 +155,12 @@ messages:
   - '&742. The answer is 42.'
   - '&7A Walshy a day will keep the troubles away.'
   - '&7Never dig straight down!'
+  - '&7Tis but a flesh wound!'
+  - '&7I'm &4NOT &7the Messiah!'
```
Comment (WalshyDev) [inline src/main/resources/languages/messages_en.yml:159]: You need to "escape" the single quote. So, do `''` (two back to back)

**Comment by WalshyDev** [comment]
Cookie, we need to run this through a YAML validator if a YAML file is edited.

**Comment by TheBusyBiscuit** [comment]
@WalshyDev I agree, workflow file or unit test?

**Comment by WalshyDev** [comment]
Definitely workflow

**Comment by TheBusyBiscuit** [comment]
Why definitely?

**Comment by WalshyDev** [comment]
Well, unit test we'd need to recursively check the resources and stuff. For a workflow we can just see if a changed file is YAML and validate that. It's a lot lighter, makes the checks fail in a more verbose way and also would validate issues in other workflow files.

**Comment by TheBusyBiscuit** [comment]
Fair, workflow validation would also be cool.
Feel free to dig one up and make a PR, would gladly add that then.

File: src/main/resources/languages/messages_en.yml [hunk 2]
```diff
@@ -155,6 +155,12 @@ messages:
   - '&742. The answer is 42.'
   - '&7A Walshy a day will keep the troubles away.'
   - '&7Never dig straight down!'
+  - '&7Tis but a flesh wound!'
+  - '&7I am &4NOT &7the Messiah!'
+  - '&7Always look on the bright side of life!'
+  - '&7African or European Swallow?'
```
Comment (TheBusyBiscuit) [inline src/main/resources/languages/messages_en.yml:161]: I would probably not include this one.

File: src/main/resources/languages/messages_en.yml [hunk 3]
```diff
@@ -155,6 +155,12 @@ messages:
   - '&742. The answer is 42.'
   - '&7A Walshy a day will keep the troubles away.'
   - '&7Never dig straight down!'
+  - '&7Tis but a flesh wound!'
+  - '&7I am &4NOT &7the Messiah!'
```
Comment (TheBusyBiscuit) [inline src/main/resources/languages/messages_en.yml:159]: Also wouldn't exactly include this either.

File: src/main/resources/languages/messages_en.yml (hunk 2 above)
Comment (Neon72) [reply src/main/resources/languages/messages_en.yml:161]: you marked all 4 which one out of the 4 you mean?

Comment (TheBusyBiscuit) [reply src/main/resources/languages/messages_en.yml:161]: I only marked one, the last one. All above is only automatically added context by GitHub.

Comment (Neon72) [reply src/main/resources/languages/messages_en.yml]: I figured 5 seconds after

**Pull request merged**